index.html

```diff
@@ -885,6 +856,158 @@ <h5>Timing Properties</h5>
+                      <td>For example, significant sounds, such as a door being slammed in anger.</td>
+                    </tr>
+                    <tr>
+                      <td><code>visual</code></td>
+                      <td>Provisional</td>
+                      <td>Parts of the visual image</td>
+                      <td>Audio Description</td>
+                      <td>See sub-types.</td>
+                    </tr>
+                    <tr>
+                      <td><code>visual;dialogue</code></td>
+                      <td>Provisional</td>
+                      <td>Verbal communication, in sign language</td>
+                      <td>Dubbing or Audio Description, translation and hard of hearing subtitles and captions, pre- and post- production scripts</td>
+                      <td>For example, a spoken conversation.</td>
+                    </tr>
+                    <tr>
+                      <td><code>visual;nonText</code></td>
+                      <td>Provisional</td>
+                      <td>Parts of the visual image that are not textual</td>
+                      <td>Audio Description</td>
+                      <td>For example, a significant object in the scene</td>
+                    </tr>
+                    <tr>
+                      <td><code>visual;text</code></td>
+                      <td>Provisional</td>
+                      <td>Textual parts of the visual image</td>
+                      <td>Audio Description</td>
+                      <td>For example, a signpost, a clock, a newspaper headline, an instant message etc.</td>
+                    </tr>
+                    <tr>
+                      <td><code>visual;text;title</code></td>
+                      <td>Provisional</td>
+                      <td>A <a>sub-type</a> of <code>visual;text</code> where the text is the title of the related media.</td>
+                      <td>Audio Description</td>
+                      <td></td>
+                    </tr>
+                    <tr>
+                      <td><code>visual;text;credit</code></td>
+                      <td>Provisional</td>
+                      <td>A <a>sub-type</a> of <code>visual;text</code> where the text is a credit, e.g. the name of an actor.</td>
+                      <td>Audio Description</td>
+                      <td></td>
+                    </tr>
+                    <tr>
+                      <td><code>visual;text;location</code></td>
+                      <td>Provisional</td>
+                      <td>A <a>sub-type</a> of <code>visual;text</code> where the text indicates the location where the content is occurring.</td>
+                      <td></td>
+                      <td></td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+              <p>User-defined values MUST begin with <code>x-</code>.</p>
```

Comment:
So user-defined values for a visual part would have to be `x-visual;myValue`? Wouldn't it be better to allow `visual;x-myValue`?

Reply:
Oh interesting, yeah I guess prefixed tokens might be what we need. Will think about that.

Reply:
Happy to allow new values to be permitted that have `x-` extension tokens appended to existing registry entries.

Reply:
Done in 224440f